src/tools.rs

```diff
@@ -744,6 +805,79 @@ Hop: From: hq5.example.org; By: hq5.example.org; Date: Mon, 27 Dec 2021 11:21:22
         );
     }
 
+    mod truncate_by_lines {
```

Comment:
just want to understand: why is `mod` needed here? i have not seen this being used in tests up to now.

Reply:
to group the tests that you can tell cargo to only run all truncate_by_lines tests